static/app/views/performance/cache/tables/transactionsTable.tsx

```diff
@@ -49,6 +60,11 @@ const COLUMN_ORDER: Column[] = [
     name: t('Project'),
     width: COL_WIDTH_UNDEFINED,
   },
+  {
+    key: `avg(${CACHE_ITEM_SIZE})`,
+    name: t('Avg Value Size'),
```

Comment:
should this be called `Avg Item Size` instead?

Reply:
Yeah i see the confusion here, It should be value size, cause that's the term you typically use for caching (key/value). The SDK sends it as `item_size` to we have to mix the terms unfortunately.